modules/serverless-inmemorychannel.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+//  * serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-inmemorychannel_{context}"]
+= Using the default InMemoryChannel configuration
+
+InMemoryChannels are for development use only, and should not be used in a production environment.
+
+The following are limitations of InMemoryChannel Channels:
+
+* No event persistence is available. If a Pod goes down, events on that Pod will be lost.
+* InMemoryChannel Channels do not implement event ordering, so two Events that are received in the Channel at the same time may be delivered to a Subscriber in any order.
+* If a Subscriber rejects an Event, there are no redelivery attempts. Instead, the rejected Event is sent to a _dead letter sink_ if this sink exists, or is otherwise dropped.
+
+When you install Knative Eventing, the following custom resource definition is created automatically:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: knative-eventing
+  name: config-br-default-channel
+data:
+  channelTemplateSpec: |
+    apiVersion: messaging.knative.dev/v1
+    kind: InMemoryChannel
+----
+// TODO: Add link for dead letter sink docs once these are added.
+
+.Creating a Channel using the cluster default configuration
+
+* Create a generic Channel custom object.
++
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1
+kind: Channel
+metadata:
+  name: example-channel
+  namespace: default
+----
++
+When the above Channel object is created, a mutating admission webhook adds a set of `spec.channelTemplate` properties for the Channel object based on the default Channel implementation.
++
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1
+kind: Channel
+metadata:
+  name: example-channel
+  namespace: default
+spec:
+  channelTemplate:
+    apiVersion: messaging.knative.dev/v1
+    kind: InMemoryChannel
+----
+
+The Channel controller will then create the backing Channel instance based on that `spec.channelTemplate` configuration. The `spec.channelTemplate` properties cannot be changed after creation, because they are set by the default channel mechanism rather than by the user.
+
+When this mechanism is used, two objects are created, a generic Channel, and an InMemoryChannel Channel.
```

Comment:
What do you think about using a colon here? Maybe something like this:

`When this mechanism is used, two objects are created: a generic Channel and an InMemoryChannel Channel.`